src/autogluon/bench/cloud/aws/batch_stack/stack.py

```diff
@@ -146,6 +146,13 @@ def __init__(self, scope: Construct, id: str, static_stack: StaticResourceStack,
             vpc=vpc,
         )
 
+        # Add inbound rule for ssh access
```

Comment:
Shall we remove this?

Reply:
I meant to keep this in comment for future use. The plan is to add an ssh access option for testing, but I'm fine to remove it in this PR.

Reply:
Sure. We can keep it then